server/src/auth.js

```diff
@@ -205,10 +205,11 @@ function mayReshapeDsCredentials(creds) {
 	}
 }
 
-// TODO: should create a checker function for each route that may be protected
+// TODO: should create a checker function for each route group that may be protected
 const protectedRoutes = {
 	termdb: ['matrix'],
-	samples: ['singleSampleData', 'getAllSamples', 'scatter', 'convertSampleId', 'getSamplesByName']
+	samples: ['singleSampleData', 'getAllSamples', 'scatter', 'convertSampleId', 'getSamplesByName'],
+	minSampleSize: ['/termdb/barsql', 'matrix', 'cuminc', 'survival', 'regression', 'scatter']
```

Comment:
@gavrielm these server routes correspond to charts that we expect to have a min sample requirement, and not to other routes like `/termdb/categories` that break filter construction if a min sample size is enforced